<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotter_image</class>
 <widget class="QWidget" name="plotter_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QFrame" name="frmWithImage">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lblViewImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveImage">
           <property name="text">
            <string>Save Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLoadImage">
           <property name="text">
            <string>Load Image</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>190</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblFormat">
          <property name="text">
           <string>Pixel format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxFormat">
          <item>
           <property name="text">
            <string>RGB_U8U8U8                  ([0,255], [0,255], [0,255])</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RGB_U8U8U8U8             ([0,255], [0,255],[0,255], 255)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RGB_F32F32F32              ([0.0f, 1.0f], [0.0f, 1.0f], [0.0f, 1.0f])</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GRAY_U8                         ([0,255])</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GRAY_F32                        ([0.0f, 1.0f])</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GRAY_F32_UNNORM     ([0.0f, 255.0f])</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GRAY_F64                        ([0.0, 1.0])</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GRAY_F64_UNNORM     ([0.0, 255.0])</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblIncomeOrder">
          <property name="text">
           <string>Order of income image lines</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxIncomeOrder">
          <item>
           <property name="text">
            <string>BOTTOM TO TOP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TOP TO BOTTOM</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblImageScale">
          <property name="text">
           <string>Image scaling factor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spbImageScale">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderScale">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblImageSize">
          <property name="text">
           <string>Image size in viewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtImageSizeInViewer">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="lblIncomeImageSize">
          <property name="text">
           <string>Income image size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtIncomeImageSize">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblUpdateTime">
          <property name="text">
           <string>Last update time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtLastUpdateTime">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxChannels">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Channels</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="cbxRed">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Red</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbxGreen">
           <property name="text">
            <string>Green</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbxBlue">
           <property name="text">
            <string>Blue</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxImageHistory">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Image History</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="lblHistoryLimit">
             <property name="text">
              <string>History limit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spbHistoryLimit">
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="lstWidgetItems">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Select items which you want to delete and then press del button</string>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="resizeMode">
            <enum>QListView::Adjust</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCleanAllHistory">
           <property name="text">
            <string>Clean all image history</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
